api/osrd_infra/views/train_schedule/standalone_simulation_report.py

```diff
@@ -64,6 +66,9 @@ def convert_simulation_results(
     route_begin_occupancy, route_end_occupancy = convert_route_occupancies(
         simulation_result["route_occupancies"], projection_path_payload, departure_time
     )
+    route_aspects = project_signal_updates(
+        simulation_result["signal_updates"], projection_path_payload, departure_time
+    )
```

Comment:
I suppose this will replace `route_begin_occupancy` and `route_end_occupancy`?

Reply:
We could keep both, the occupancies can be used as a kind of "bounding box" that will be useful when handling overlaps. This new set of squares isn't the most convenient for that.